### ISSUE TITLE
add warning if # of sims is below 10 at top level

### ISSUE THREAD

**Issue by graemeblair** [DeclareDesign/DeclareDesign#281]
i.e. should be triggered here:

```
library(DeclareDesign)
U <- declare_population(
  world = add_level(N = 1, tau = sample(0:1,1)),
  samp  = add_level(N = 10, Y_Z_0 = rnorm(N)/100, Y_Z_1 = 1)
)

Z <- declare_assignment()

R <- declare_reveal()

Q <- declare_estimand(inq = mean(tau))

A <- declare_estimator(Y~Z, estimand = "inq")


des <- U + Z + R + Q + A 

set.seed(1)
diagnose_design(des,  sims = c(1, 100, 1, 1, 1))

set.seed(2)
diagnose_design(des,  sims = c(1, 100, 1, 1, 1))
```

**Comment by macartan** [comment]
Maybe only if bootstrap sims > 0

**Comment by nfultz** [comment]
Lets do 30 instead of 10

**Comment by graemeblair** [comment]
Think we should make warning if top level is <30 AND if the product of the fanout (total # of sims) is <30

**Comment by graemeblair** [comment]
closed by #288
